drivers/src/memory_layout.rs

```diff
@@ -74,7 +75,8 @@ pub const DPE_SIZE: u32 = 5 * 1024;
 pub const PCR_RESET_COUNTER_SIZE: u32 = 1024;
 pub const AUTH_MAN_IMAGE_METADATA_MAX_SIZE: u32 = 7 * 1024;
 pub const IDEVID_CSR_SIZE: u32 = 1024;
-pub const DATA_SIZE: u32 = 27 * 1024;
```

Comment:
nit: now that https://github.com/chipsalliance/caliptra-sw/pull/1722 is merged, should be able to just add to the PersistentData driver and leave memory layout as-is